test/unit/base_cli_test.rb

```diff
@@ -7,6 +7,36 @@
 describe 'BaseCLI' do
   let(:cli) { Inspec::BaseCLI.new }
 
+  describe 'opts' do
+    it 'raises if `--password/--sudo-password` are used without value' do
+      default_options = { mock: { sudo_password: -1 } }
+      Inspec::BaseCLI.stubs(:default_options).returns(default_options)
+
+      e = proc { cli.send(:opts, :mock) }.must_raise(ArgumentError)
+      e.message.must_match /Please provide a value for --sudo-password/
+    end
+
+    it 'assumes `--sudo` if `--sudo-password` is used without it' do
+      default_options = { mock: { sudo_password: 'p@ssw0rd' } }
```

Comment:
yep, no matter how many times you type p@ssw0rd, it will show to us as ********

Reply:
Gah! I missed my chance to use hunter2.